### DISCUSSION TITLE
post link

### DISCUSSION THREAD

**Discussion by dennis8989** [TLYu0419/facebook_crawler#3] [Q&A] [answered]
請問產出的報表是否有link可以直接連到想看的post的網頁呢

**Comment by tlyu0419** [comment, marked as answer]
Hi  @dennis8989 
請將文章的ID前面加上 https://www.facebook.com/ 的字串就是貼文的網址了
以文章ID '123456789' 為例，貼文的連結即是 https://www.facebook.com/123456789
